src/Controller/ProfileController.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
i think Profile is not enough to be a Controller

Reply:
For me I think it can be a controller